tsuru/resource_tsuru_cluster.go

```diff
@@ -0,0 +1,147 @@
+package tsuru
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/tsuru/go-tsuruclient/pkg/tsuru"
+)
+
+func resourceTsuruCluster() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceTsuruClusterCreate,
+		ReadContext:   resourceTsuruClusterRead,
+		UpdateContext: resourceTsuruClusterUpdate,
+		DeleteContext: resourceTsuruClusterDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
We should add `Description` to all schema items.

Reply:
yeap @pedrokiefer, let's add after merge